examples/k8s_sa.py

```diff
@@ -0,0 +1,45 @@
+"""This example showcases the hello world example of Hera using k8s service account token"""
+from hera.v1.task import Task
+from hera.v1.workflow import Workflow
+from hera.v1.workflow_service import WorkflowService
+
+from kubernetes import client, config
+import base64
+
+
+def get_sa_token(service_account: str, namespace: str = "default", config_file=None):
```

Comment:
I am aware this is an example but curious what you think about adding a type for the `config_file` field as a `str`, perhaps, or `Path`, and checking for its existence to throw an exception if the file is not there? I think it might help users out by covering a potential failure case.

Reply:
yes, you are right it should be `Optional[str]`, the [expected input is 'str'](https://github.com/kubernetes-client/python-base/blob/master/config/kube_config.py#L48).. fixing it!